Library/Formula/commandbox.rb

```diff
@@ -0,0 +1,45 @@
+class Commandbox < Formula
+  desc "CFML embedded server, package manager, and app scaffolding tools"
+  homepage "http://www.ortussolutions.com/products/commandbox"
+  url "http://downloads.ortussolutions.com/ortussolutions/commandbox/3.0.0/commandbox-bin-3.0.0.zip"
+  sha256 "194738f683cbbbaa63f7a2ae5d1fe6c0d09ef8370c4ac0b6c9aa087f6013abc1"
+  devel do
+    url "http://integration.stg.ortussolutions.com/artifacts/ortussolutions/commandbox/3.1.0/commandbox-bin-3.1.0.zip"
+    sha256 "0ba0bb96114a73961ee4a8e3a656c72b98cd2f3cd561b92d159bebcfef240418"
+  end
+
+  depends_on :arch => :x86_64
```

Comment:
@mistydemeo :  Thanks for the question.  I'm removing the arch requirement in the next push. An earlier iteration had a compiled source component, but the current version is all java.